src/objects/Requirement__c.object

```diff
@@ -123,6 +123,14 @@
         <trackTrending>false</trackTrending>
         <type>Lookup</type>
     </fields>
+    <fields>
+        <fullName>Identifier__c</fullName>
+        <displayFormat>R-{0000000}</displayFormat>
```

Comment:
Quick note -- Disbursement's auto-number is currently using 5 digits rather than 6 if we wanted to be consistent.